src/InstrumentationEngine.Attach/InstrumentationConfigurationSources.xsd

```diff
@@ -0,0 +1,53 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<xs:schema attributeFormDefault="unqualified" elementFormDefault="qualified" xmlns:xs="http://www.w3.org/2001/XMLSchema">
+    <xs:element name="InstrumentationConfigurationSources">
+        <xs:complexType>
+            <xs:sequence>
+                <xs:element name="InstrumentationConfigurationSource" maxOccurs="10">
```

Comment:
> maxOccurs="10" [](start = 69, length = 15)

Do we really need any of these maxOccurs?

Reply:
By default they're 1, and the alternative is to set it as `unbounded`. I chose 10 arbitrarily.

---
In reply to: [387829964](https://github.com/microsoft/CLRInstrumentationEngine/pull/220#discussion_r387829964) [](ancestors = 387829964)

Reply:
Let's use unbounded.

---
In reply to: [387832091](https://github.com/microsoft/clrinstrumentationengine/pull/220#discussion_r387832091) [](ancestors = 387832091,387829964)